src/main/java/io/strimzi/kafka/bridge/tracing/OpenTracingHandle.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.tracing;
+
+import io.jaegertracing.Configuration;
+import io.opentracing.References;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.kafka.TracingKafkaUtils;
+import io.opentracing.contrib.kafka.TracingProducerInterceptor;
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMap;
+import io.opentracing.propagation.TextMapAdapter;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+import io.strimzi.kafka.bridge.config.BridgeConfig;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Properties;
+
+import static io.strimzi.kafka.bridge.tracing.TracingConstants.COMPONENT;
+import static io.strimzi.kafka.bridge.tracing.TracingConstants.KAFKA_SERVICE;
+
+/**
+ * OpenTracing implementation of TracingHandle.
+ */
+class OpenTracingHandle implements TracingHandle {
+
+    private static Tracer tracer;
+
+    private static Tracer getTracer() {
```

Comment:
`GlobalTracer.get` is just returning "the constant GlobalTracer", so I don't think we need the static field and associated null check. Maybe we don't need this method either, unless we're trying to hide from the call sites of this method that it's the global trace which we're using.